data/cookbook/web-programming/uri-parsing/00-uri.ml

```diff
@@ -0,0 +1,44 @@
+---
+packages:
+- name: "uri"
+  version: "4.2.0"
+libraries:
+  - uri
+discussion: |
+  - **Understanding `Uri`:** `Uri` provides useful functions for parsing and encoding URI (URL or URN). It can be handy with complex URI (parameters, special or non ASCII characters)
```

Comment:
Same question in li.8 of the previous document.

Reply:
Same correction applied